tests/test_client.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+import testinfra
+import subprocess
+import os
+import time
+import json
+
+
+@pytest.fixture
+def entities():
+    return {
+        'organization_label': 'Test_Organization',
+        'activation_key': 'Test AK',
+        'product': 'Test Product',
+        'product_label': 'Test_Product',
+        'yum_repository': 'Zoo',
+        'yum_repository_label': 'Zoo',
+        'errata_id': 'RHEA-2012:0055',
+        'container_repository_label': 'foremanbusybox'
+    }
+
+@pytest.fixture(scope='session')
+def client(request):
+    name = "smoker-client-%s" % request.param.replace('/', '_').replace(':', '_')
+    podman_id = subprocess.check_output(['podman', 'ps', '-q', '--filter', "name=%s" % name])
+
+    if not podman_id:
+        subprocess.check_output(['podman', 'run', '--name', name, '-dt', request.param]).decode().strip()
+
+    # return a testinfra connection to the container
+    yield testinfra.get_host("podman://" + name)
+
+@pytest.fixture(scope='session')
+def smoker():
+    return testinfra.get_host("local://localhost")
```

Comment:
Represents the smoker box that commands like hammer are ran on.

Reply:
In my test I could use `local://` as a connection string.